libagent/formats.py

```diff
@@ -57,10 +59,39 @@ def parse_pubkey(blob):
 
     result = {'blob': blob, 'type': key_type, 'fingerprint': fp}
 
-    if key_type == SSH_NIST256_KEY_TYPE:
+    if key_type == SSH_NIST256_KEY_TYPE or key_type == SSH_NIST256_CERT_TYPE:
+        if key_type == SSH_NIST256_CERT_TYPE:
+            # nonce
+            _ = util.read_frame(s)
+
         curve_name = util.read_frame(s)
         log.debug('curve name: %s', curve_name)
         point = util.read_frame(s)
+
+        if key_type == SSH_NIST256_CERT_TYPE:
```

Comment:
Please add a unit test for the new code.

Reply:
Do you mean `assert`ions?

Reply:
> Do you mean `assert`ions?

Similar to how it's done here:
https://github.com/romanz/trezor-agent/blob/f0769655adefbd9c1f06beffcc6fb7ff16d71834/libagent/tests/test_formats.py#L27
https://github.com/romanz/trezor-agent/blob/f0769655adefbd9c1f06beffcc6fb7ff16d71834/libagent/tests/test_formats.py#L44

Reply:
Done